src/Tribe/Aggregator/Event.php

```diff
@@ -189,7 +189,7 @@ public function get_existing_ids( $origin, $values ) {
 
 		$key = "_{$fields[ $origin ]['target']}";
 		$post_type = Tribe__Events__Main::POSTTYPE;
-		$interval = "('" . implode( "','", array_map( 'absint', $values ) ) . "')";
+		$interval = "('" . implode( "','", $values ) . "')";
```

Comment:
What are your thoughts on keeping the `array_map()` but with `esc_sql` as the callback?

Reply:
I've added it, good idea.